models/Amity.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/python
```

Comment:
I see you have all your program files inside the models folder, I think you misunderstood the purpose of models. You should create a 'Views' folder for all of these files except your db models file.